src/lib/intent/base-intent.ts

```diff
@@ -155,6 +155,22 @@ export default abstract class Intent implements IntentInterface {
     return this.type
   }
 
+  getTagId (): string {
+    return ''
+  }
+
+  isTagCreate (): boolean {
+    return false
+  }
+
+  isTagUpdate (): boolean {
+    return false
+  }
+
+  isTagIntent (): boolean {
```

Comment:
We should really refactor things to get rid of this pattern in the future.  
An intent nowadays has more properties that say what it is not, than what it is :)

Reply:
Just a note to self for future changes though, nothing to do right now :)

Reply:
:) I agree. But to get rid of this we would need different base-intents for different entities I guess ...